source/MRMesh/MRObjectLabel.h

```diff
@@ -0,0 +1,100 @@
+#pragma once
+#include "MRVisualObject.h"
+#include "MRSymbolMesh.h"
+
+namespace MR
+{
+
+
+/// This object type render label in scene
+/// \ingroup DataModelGroup
+class MRMESH_CLASS ObjectLabel : public VisualObject
+{
+public:
+    MRMESH_API ObjectLabel();
+
+    ObjectLabel( ObjectLabel&& ) noexcept = default;
+    ObjectLabel& operator = ( ObjectLabel&& ) noexcept = default;
+    virtual ~ObjectLabel() = default;
+
+    constexpr static const char* TypeName() noexcept
+    {
+        return "ObjectLabel";
+    }
+    virtual const char* typeName() const override
+    {
+        return TypeName();
+    }
+
+    MRMESH_API virtual void applyScale( float scaleFactor ) override;
+
+    MRMESH_API virtual std::shared_ptr<Object> clone() const override;
+    MRMESH_API virtual std::shared_ptr<Object> shallowClone() const override;
+
+    /// sets size of label font on screen in pixels
+    virtual void setFontHeight( float size )
+    {
+        fontHeight_ = size;
+    }
+    /// returns size of label font on screen in pixels
+    float getFontHeight() const
+    {
+        return fontHeight_;
+    }
+
+    /// sets text and position of label
+    MRMESH_API void setLabel( const MeshLabel& label );
```

Comment:
Rename MeshLabel in PositionedText ? (Keep MeshLabel alias for a while as deprecated)